platform/security/certificate/src/main/java/org/codice/ddf/certificate/JKSManager.java

```diff
@@ -0,0 +1,207 @@
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JKSManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JKSManager.class);
+
+    /*---------------------------------------------------------------------
+    |  Method: saveCertToJKSKeyStore
+    |
+    |  Purpose:  Saves a certificate to an existing keystore file,
+    |
+    |  Pre-condition:  A valid keystore file
+    |
+    |  Post-condition: The keystore file will contain the certificate passed in a parameter (certficateChain)
+    |
+    |  Parameters:
+    |        String  keyStorePath    -- path to the keyStore
+    |        String  keyStorePassword   -- password for the keystore
+    |        String  alias           -- alias for the new keystore entry
+    |        String keyPassword      -- password to protect the key
+    |        X509Certificate[] certChain -- the certificate chain for the corresponding public key
+    |        PrivateKey privKey      -- privateKey to be associated with the alias
+    |
+    |  Returns:  None
+    *-------------------------------------------------------------------*/
+
+    public static void addCertToKeyStore(String keyStorePath, String keyStorePassword, String alias,
+            String keyPassword, X509Certificate[] certChain, PrivateKey privKey) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
+            jks.load(new FileInputStream(keyStorePath), keyStorePassword.toCharArray());
+            jks.setKeyEntry(alias, privKey, keyPassword.toCharArray(), certChain);
+            jks.store(new FileOutputStream(keyStorePath), keyStorePassword.toCharArray());
+        } catch (CertificateException | NoSuchAlgorithmException | IOException | KeyStoreException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * ---------------------------------------------------------------------
+     * Method: deleteCertFromKeyStore
+     * <p/>
+     * Purpose:  Deletes the certificate with the given alias from the keystore.
+     * <p/>
+     * Pre-condition:  Keystore contains a certificate with the given alias
+     * <p/>
+     * Post-condition: The keystore file that is output will contain the certificate passed in as a parameter
+     *
+     * @param keyStorePath     path to the keyStore
+     * @param keyStorePassword password for the keystore
+     * @param alias            alias of the certificate to delete
+     *                         -------------------------------------------------------------------
+     */
+
+    public static void removeCertFromKeyStore(String keyStorePath, String keyStorePassword,
+            String alias) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
+            jks.load(new FileInputStream(keyStorePath), keyStorePassword.toCharArray());
+            jks.deleteEntry(alias);
+            jks.store(new FileOutputStream(keyStorePath), keyStorePassword.toCharArray());
```

Comment:
These streams should be in a "try with resources" so that they can get closed in the event of an exception occurring during loading or storing. Otherwise, they'll be left open.